src/identity/components/IdentityProvider.tsx

```diff
@@ -15,13 +16,15 @@ type IdentityProvider = {
   address?: Address;
   children: ReactNode;
   schemaId?: Address | null;
+  chain?: Chain;
```

Comment:
Let's move this type into the `types` file, and keep the keys alphabetically ordered.